great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -53,8 +53,9 @@ def __init__(  # noqa: PLR0913
         self.forbidden_substrings = forbidden_substrings
         self.platform_specific_separator = platform_specific_separator
 
-        if filepath_template is not None and filepath_suffix is not None:
-            raise ValueError("filepath_suffix may only be used when filepath_template is None")  # noqa: TRY003
+        if filepath_template and filepath_suffix:
```

Comment:
I'd log an info message here saying both were specified so we are taking template.